app/src/components/common/Button.vue

```diff
@@ -9,24 +9,29 @@
 </template>
 
 <script>
-export default {
+import Component from 'vue-class-component'
+import Vue from 'vue'
+
+@Component({
   props: {
```

Comment:
Please use Vue Property Decorator (`@Prop`)  for props, link for more info: https://blog.logrocket.com/define-properties-vue-property-decorator-typescript/

Reply:
Thank you for advice! Defined props using Vue Property Decorator in recent commit.